documentation/features/forecasting.rst

```diff
@@ -46,32 +67,33 @@ Here is how you can ask for forecasts to be made in the CLI:
 
 .. code-block:: bash
 
-    flexmeasures add forecasts --from-date 2024-02-02 --to-date 2024-02-02 --horizon 6 --sensor 12  --as-job
+    flexmeasures add forecasts --from-date 2024-02-02 --to-date 2024-02-02 --horizon 6 --sensor 12  --as-job  # add train-start
 
 Sensor 12 would represent the power readings of your solar power, and here you ask for forecasts for one day (2 February, 2024), with a forecast of 6 hours.
 
 The ``--as-job`` parameter is optional. If given, the computation becomes a job which a worker needs to pick up. There is some more information at :ref:`how_queue_forecasting`.
 
 
-Rolling vs fixed-point
--------------------------
-
-These forecasts are `rolling` forecasts ― which means they all have the same horizon. This is useful mostly for analytics and simulations.
+Fixed-point vs rolling
+----------------------
 
-We plan to work on fixed-point forecasts, which would forecast all values from one point in time, with a growing horizon as the forecasted time is further away.
-This resembles the real-time situation better.
+Unlike previous rolling forecasts, the new infrastructure generates **fixed-point forecasts**:
```

Comment:
I believe you explained this well already. We could explain what rolling forecasts are here (using the old explanation about sharing the same horizon). It is not that crucial, just a clarification.